io/subsystem/src/main/java/org/wildfly/extension/io/WorkerResourceDefinition.java

```diff
@@ -77,7 +77,7 @@
             .build();
     static final OptionAttributeDefinition WORKER_TASK_KEEPALIVE = new OptionAttributeDefinition.Builder(Constants.WORKER_TASK_KEEPALIVE, Options.WORKER_TASK_KEEPALIVE)
             .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
-            .setDefaultValue(new ModelNode(60))
+            .setDefaultValue(new ModelNode(60_000))
```

Comment:
added transformers. They ware also needed for expressions that ware added as part of this commit.